docs/jwst/associations/generator.rst

```diff
@@ -52,20 +50,18 @@ have. For example, if a JWST coronagraphic observation was performed, but the
 related PSF observation failed, the coronagraphic association would be marked invalid.
 
 Once validation is complete, :py:func:`~jwst.associations.generate` returns a
-2-tuple. The first item is a list of the associations created. The second item
-is another :py:class:`~jwst.associations.AssociationPool` containing all the
-members that did not get added to any association.
+list of the associations created.
 
 .. _member-with-lists:
 
 Member Attributes that are Lists
 --------------------------------
 
-As mentioned in :ref:`design-pool`, most member attributes are simply
```

Comment:
"simply" does not add value to the narrative and can have negative impact pedagogically.